recipe/meta.yaml

```diff
@@ -11,16 +11,16 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
+  number: 2
 
 outputs:
   - name: rust_{{ cross_target_platform }}
     script: build.sh  # [unix]
     script: bld.bat  # [win]
     build:
       skip: True  # [win and vc<14]
-      # TODO: need more work to pass the LDFLAGS and need to use clang-cl
-      skip: True  # [cross_target_platform == "win-64" and unix]
+      # TODO: need more work to separate build and host toolchain on osx
```

Comment:
What does this mean?

Reply:
I didn't try to configure the windows target correctly for cross compiles from osx, it looked like some environment variables had to be adjusted which could make a follow up PR.

Reply:
Should be fixed with https://github.com/conda-forge/clang-win-activation-feedstock/pull/27